src/quacc/recipes/mrcc/core.py

```diff
@@ -0,0 +1,67 @@
+"""Core recipes for MRCC."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from quacc import job
+from quacc.recipes.mrcc._base import run_and_summarize
+
+if TYPE_CHECKING:
+    from ase.atoms import Atoms
+
+    from quacc.types import Filenames, RunSchema, SourceDirectory
+
+
+@job
+def static_job(
+    atoms: Atoms,
+    charge: int = 0,
+    spin_multiplicity: int = 1,
+    method: str = "pbe",
+    basis: str = "sto-3g",
+    mrccinput: dict[str, str] | None = None,
+    mrccblocks: str | None = None,
+    copy_files: SourceDirectory | dict[SourceDirectory, Filenames] | None = None,
+) -> RunSchema:
+    """
+    Carry out a single-point calculation.
+
+    Parameters
+    ----------
+    atoms
+        Atoms object
+    charge
+        Charge of the system.
+    spin_multiplicity
+        Multiplicity of the system.
+    method
+        The method [e.g., PBE or CCSD(T)] to use, this is the value for the calc keyword.
+    basis
+        Basis set
+    mrccinput
+        Dictionary of `mrccinput` swaps for the calculator. To remove entries
+        from the defaults, put a `#` in front of the name. Refer to the
+        [quacc.calculators.mrcc.mrcc.MRCC][] calculator for details on `mrccinput`.
+    mrccblocks
+        String for the `mrccblocks` input.
+    copy_files
+        Files to copy (and decompress) from source to the runtime directory.
+
+    Returns
+    -------
+    RunSchema
+        Dictionary of results from [quacc.schemas.ase.Summarize][]
+    """
+    default_inputs = {"calc": method, "basis": basis}
```

Comment:
According to the manual, RKS is a synonym for RHF and similarly for UKS and UHF. The type of calculation is controlled by the `scftype` keyword as described in this manual: https://mrcc.hu/MRCC/manual/pdf/manual.pdf. There is also the choice of restricted open-shell calculations through ROHF and ROKS. UHF/UKS and ROHF/ROKS can both be used for CCSD(T), so I think this doesn't need to be coupled to the method. Given all this, I think we can make this simply check if spin_multiplicity >  1 and raise a `ValueError` if scftype is not provided or equal to RHF/RKS. Does that sound reasonable?

Reply:
Where would this best be implemented in `prep_calculator` or `run_and_summarize` of _base or in `static_job` or core? I'm thinking in prep_calculator after 
`    mrccinput = recursive_dict_merge(default_inputs, input_swaps)`

Reply:
Assuming this would apply for all recipes (seems likely) then I agree with your decision about prep_calculator. I also think raising the error is wise.